web_notify/__manifest__.py

```diff
@@ -6,9 +6,11 @@
     'name': 'Web Notify',
     'summary': """
         Send notification messages to user""",
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
+    'description': 'Web Notify',
     'license': 'AGPL-3',
-    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'author': """Serpent Consulting Services Pvt. Ltd.,
+                ACSONE SA/NV, Odoo Community Association (OCA)""",
```

Comment:
Just a remark: I know there is no clear rule for that but I tend to think it's not fair to add your company as author when the work is a trivial migration like this one. It's perfectly fine to add you in the contributors list, though. What do you think?